client/root.js

```diff
@@ -50,12 +51,23 @@ async function handleLoggedIn( context ) {
 	const state = context.store.getState();
 	const siteSlug = getSiteSlug( state, primarySiteId );
 	const isCustomerHomeEnabled = canCurrentUserUseCustomerHome( state, primarySiteId );
+	const launchpadTasksComplete = false;
+	const launchpadFlow = 'newsletter';
 
 	let redirectPath;
 
 	if ( ! siteSlug ) {
 		// there is no primary site or the site info couldn't be fetched. Redirect to Reader.
 		redirectPath = '/read';
+	} else if (
+		isCustomerHomeEnabled &&
+		! launchpadTasksComplete &&
+		config.isEnabled( 'signup/launchpad' )
+	) {
+		// The new stepper launchpad onboarding flow isn't registered within the "page"
+		// client-side router, so page.redirect won't work. We need to use the
+		// traditional window.location Web API.
+		redirectToLaunchpad( siteSlug, launchpadFlow );
 	} else if ( isCustomerHomeEnabled ) {
 		redirectPath = `/home/${ siteSlug }`;
```

Comment:
If this latter pre-existing condition is sending to MyHome and we are adding the `maybeRedirect` in myHome, technically that will handle this case as well? Im assuming the reason to also add a redirect here is for this flow to be a little more seamless and not add an extra load to the process? Or is there another reason?

Reply:
> If this latter pre-existing condition is sending to MyHome and we are adding the maybeRedirect in myHome, technically that will handle this case as well? Im assuming the reason to also add a redirect here is for this flow to be a little more seamless and not add an extra load to the process? Or is there another reason?

That's what I would've assumed,  but there are two instances of the call to redirect `page.redirect( `/stats/day/${ slug }` );` in `maybeRedirect` and in line 60 of this file as well.